.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Question
+  - name: Discussions
+    url: https://github.com/hikari-py/hikari/discussions
+    about: A perfect place to ask questions and get help for issues you are having.
```

```suggestion
    about: A perfect place to ask questions and get help with issues you are having.
```
sorry for not making this suggestion later but this should be the last one, promise